docs/javascript/v2/features/simulcast.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Simulcast
+nav: 3.33
+---
+
+## Minimum requirements
```

Comment:
currently this section is bleeding into the introduction. can we separate them out?